### PR TITLE
overrides get command

### PR DESCRIPTION
**Pull request by rabiaq123**
- `dvc overrides get --feature <feature-key> --environment <environment-key>` 
- use case: viewing a feature-specific override given the environment

### CODE REVIEW

File: src/api/zodClient.ts [hunk 1]
```diff
@@ -3372,6 +3393,63 @@ const endpoints = makeApi([
...
         ],
     },
+    {
+        method: 'get',
+        path: '/v1/projects/:project/features/:feature/overrides/current',
+        alias: 'OverridesController_findOne',
+        requestFormat: 'json',
+        parameters: [
+            {
+                name: 'project',
```
Comment (kaushalkapasi) [inline src/api/zodClient.ts:3403]: missing feature param (also type `Path`) and environment param (type `Query`)

File: src/commands/overrides/get.ts [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        const overrides = await fetchFeatureOverridesForUser(this.authToken, this.projectKey, featureKey) 
+        const environment = await fetchEnvironmentByKey(this.authToken, this.projectKey, environmentKey)
+        const override = overrides.overrides.find((override) => override._environment === environment._id)
+
+        if (!override) {
+            if (headless) {
+                this.writer.showResults({ environment: environment.key, variation: null })
+                return
+            }
+            this.writer.showError(
```
Comment (kaushalkapasi) [inline src/commands/overrides/get.ts:75]: this should just be normal output, not `showError`

File: src/commands/overrides/get.ts [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+} from '../../ui/prompts'
+import Base from '../base'
+import { fetchFeatureOverridesForUser } from '../../api/overrides'
+import { fetchEnvironmentByKey } from '../../api/environments'
+import { fetchVariationByKey } from '../../api/variations'
+
+export default class DetailedTargeting extends Base {
+    static hidden = false
+    authRequired = true
+    static description = 'View the overrides associated with your DevCycle Identity in your current project.'
```
Comment (chantalchung) [inline src/commands/overrides/get.ts:17]:
```suggestion
    static description = 'View the Overrides associated with your DevCycle Identity in your current project.'
```

File: src/commands/overrides/get.ts [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+    EnvironmentPromptResult, 
+    featurePrompt, 
+    FeaturePromptResult 
+} from '../../ui/prompts'
+import Base from '../base'
+import { fetchFeatureOverridesForUser } from '../../api/overrides'
+import { fetchEnvironmentByKey } from '../../api/environments'
+import { fetchVariationByKey } from '../../api/variations'
+
+export default class DetailedTargeting extends Base {
```
Comment (chantalchung) [inline src/commands/overrides/get.ts:14]: should this be named something else?
```suggestion
export default class DetailedOverrides extends Base {
```

File: src/commands/overrides/get.ts [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+        if (!override) {
+            if (headless) {
+                this.writer.showResults({ environment: environment.key, variation: null })
+                return
+            }
+            this.writer.showResults(
+                `Override for feature: ${featureKey} on environment: ${environment.key} is variation: <not-set>`
+            )
+            this.writer.infoMessageWithCommand('To set an override, use: ', 'dvc overrides update')
```
Comment (chantalchung) [inline src/commands/overrides/get.ts:77]: space is not needed here, it's handled in the function
```suggestion
            this.writer.infoMessageWithCommand('To set an override, use:', 'dvc overrides update')
```

File: src/commands/overrides/get.ts [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+        )
+        const environment = await fetchEnvironmentByKey(this.authToken, this.projectKey, environmentKey)
+        const override = overrides.overrides.find((override) => override._environment === environment._id)
+
+        if (!override) {
+            if (headless) {
+                this.writer.showResults({ environment: environment.key, variation: null })
+                return
+            }
+            this.writer.showResults(
```
Comment (chantalchung) [inline src/commands/overrides/get.ts:74]: let's use `showRawResults` here so it doesn't stringify the result again and add extra quotations
image

Comment (suthar26) [reply src/commands/overrides/get.ts:74]: you mean line 74, just to be clear?

Comment (chantalchung) [reply src/commands/overrides/get.ts:74]: yup, in non-headless mode

**Pull request merged**